A/AcousticsToolbox/build_tarballs.jl

```diff
@@ -22,6 +22,17 @@ make
 mkdir -p $bindir
 find . -name *.exe -exec cp {} $bindir \;
 cd $WORKSPACE/srcdir/ORCA_Mode_modelling_gfortran/src
+cat > patch << EOF
+c: HACK: dump mode functions and acoustic pressure to a file
+               open(67,file='modes_tlc.bin',form='unformatted')
+               write(67) nzsr, nmode, nrec, nsrc
+               write(67) ((phi(j,jm), j=1,nzsr), jm=1,nmode)
+               write(67) ((tlc(jrec,jsrc), jrec=1,nrec), jsrc=1,nsrc)
+               close(67)
+EOF
+head -106 cw_modes.f > tmp1
```

Comment:
This is extremely brittle, when the lines of this file will shift this won't make any sense. Having a real patch, with context, is much more robust.

Reply:
Yes, I understand that, but the patch is for a specific version of the code. I will have to maintain it for newer versions - or hopefully the original author will fix the problem (I have reported it) and the patch won't be necessary.

The alternative is for me to host a copy of the code, which I prefer not to.

Do you have other suggestions?

Reply:
Ok, will take a look and update shortly. Thanks.

Reply:
Moved the changes to a `cw_modes.patch` file and applied it using `atomic_patch`.